.github/workflows/release.yaml

```diff
@@ -6,11 +6,6 @@ on:
     branches:
```

Comment:
` # Trigger the workflow on push or pull request ` 
This workflow is triggered only on the push on master.